Packages.props

```diff
@@ -4,7 +4,7 @@
 		<PackageReference Update="Autofac" Version="$(AutofacVersion)" />
```

Comment:
Could you split the file into multiple item groups like we do in other solutions while you're at it?